.circleci/config.yml

```diff
@@ -1,37 +1,25 @@
 ---
 version: 2.1
 orbs:
-  ruby-rails: sul-dlss/ruby-rails@4.4.0
-  browser-tools: circleci/browser-tools@1.5.0
-  node: circleci/node@7.0.0
```

Comment:
Now we use the cypress orb instead of the node orb to handle JS dependencies for the Cypress job.